site/en/docs/privacy-sandbox/attribution-reporting/index.md

```diff
@@ -132,8 +126,7 @@ Event-level reports associate an ad click or view with coarse conversion data.
 
 Event-level reports are suited for:
 
-- **Optimization**. Event-level reports help answer questions like "How can I improve my return on
-  investment?". In particular, these reports can be used to optimize for ad placement, since a
+- **Optimization**. Event-level reports help answer questions like "How can I improve my return on investment?". In particular, these reports can be used to optimize for ad placement, since a
```

Comment:
```suggestion
- **Optimization**. Event-level reports help answer questions like "How can I improve my return on
  investment?". In particular, these reports can be used to optimize for ad placement, since a
```